src/site/content/en/handbook/content-types/index.md

```diff
@@ -9,20 +9,20 @@ description: |
 ---
 
 ## Posts
-**Format:** primarily informational with instructions included as appropriate
+**Format:** Primarily informational with instructions included as appropriate
```

Comment:
I don't know how this is different from a codelab. Both have steps, but codelabs have more code? Seems like a gray area.

Reply:
You're detecting the inherent fuzziness. I'd say there's more of a continuum between codelabs and posts, with the one concrete difference being that codelabs have a side-by-side layout with the content on the left and Glitch on the right. I may reframe this page to say that actually :)

Reply:
We had codelabs long before web.dev. To me the difference was always that codelabs guide the user through doing while posts merely show and a firm line between was maintained by decided how the material would be presented and sticking to that plan.

Reply:
FWIW, for web.dev we use the term codelab to indicate that it will be using the two column layout with text on the left and code on the right. That's the primary distinction for me.

Reply:
That sure sounds like a practical application of my heuristic.

Reply:
I don't think that heuristic quite holds for web.dev, @jpmedley, because posts also often guide  people through doing. I think Rob's right; I might reframe this page to (a) note format conventions for all web.dev content, (b) note that we have two layouts, and (c) provide a heuristic for when to use each layout.

Reply:
'Guide' was clearly the wrong word. I was trying to capture the idea that codelabs give users instructions for completing working code and a sandbox in which to do it.  If posts provide a sandbox (i.e. samples), those sandboxes contain already completed code. Posts also may not provide a complete end-to-end implementation. For example, this would happen when a new capability is added to an existing feature.